CVE-2009-2693.yml

```diff
@@ -0,0 +1,220 @@
+CVE: CVE-2009-2693
+

```

Comment:
Please delete this file. It should be in the other directory.

Reply:
I don't know how this file got here but thank you for noticing it. I'll delete it.